modules/options/src/main/scala/scala/build/interactive/InteractiveFileOps.scala

```diff
@@ -0,0 +1,32 @@
+package scala.build.interactive
+
+object InteractiveFileOps {
+
+  def erasingPath(
+    interactive: Interactive,
+    printableDest: String,
+    destPath: os.Path
+  )(fallbackAction: () => Unit) = {
+    val msg = s"""|Destination path: $printableDest already exists.
```

Comment:
I'd phrase it this way:
```suggestion
    val msg = s"""|Destination $printableDest already exists.
```

Reply:
Or maybe even
```suggestion
    val msg = s"""|$printableDest already exists.
```